cmd/xl-v1.go

```diff
@@ -264,7 +261,9 @@ func getStorageInfo(disks []StorageAPI) StorageInfo {
 // StorageInfo - returns underlying storage statistics.
 func (xl xlObjects) StorageInfo() StorageInfo {
 	storageInfo := getStorageInfo(xl.storageDisks)
-	storageInfo.Backend.ReadQuorum = xl.readQuorum
-	storageInfo.Backend.WriteQuorum = xl.writeQuorum
+	// return the default quorum data
+	// per object disk info is available in object meta data
+	storageInfo.Backend.ReadQuorum = xl.metaReadQuorum
```

Comment:
This is actually meaningless now, we should fix it and remove these variables. Printing them doesn't make sense unless we do it correctly. This also doesn't signify "default" - default is StandardClass which is also configurable.

Reply:
Yes, agreed, let's remove these variables.